integreat_cms/api/v3/chat/user_chat.py

```diff
@@ -99,6 +105,18 @@ def get_messages(
     return response_or_error(client.get_messages(user_chat))
 
 
+def translate_message(message: str, language_slug: str) -> str:
+    """
+    Translate a string with Google Translate
+    """
+    credentials = service_account.Credentials.from_service_account_file(
+        settings.GOOGLE_APPLICATION_CREDENTIALS
+    )
+    translate_client = translate.Client(credentials=credentials)
+    result = translate_client.translate(message, target_language=language_slug)
```

Comment:
Should we track these translations and add them to the regions translation budget, or are they free for the regions?

Reply:
For the time being it is "free". We may want to host our own translation model for this purpose soon. Also something that made me think for a moment and what I had not yet considered thoroughly enough: authentication of requests for translation purposes is done via the reverse look up. I'll add a get_or_404() for resolving the Zammad server IP to a region in https://github.com/digitalfabrik/integreat-cms/pull/3051/files#diff-a9958a0ba0d1f97dd15f8eb15d9fa3723486384f9f04b0f2bd6ba7ff48a3a177R247